src/content/en/updates/2018/10/background-fetch.md

```diff
@@ -0,0 +1,448 @@
+      API</a>. Calling without arguments returns a promise for all records.</p>
+      <p>See below for more details.</p></td>
+    </tr>
+    <tr>
+      <td><code>match(request, opts)</code></td>
+      <td>Returns <code>Promise&lt;BackgroundFetchRecord&gt;</code><br>As above, but resolves with
+      the first match.</td>
+    </tr>
+    <tr><th colspan=2>Events</th></tr>
+    <tr>
+      <td><code>progress</code></td>
+      <td>Fired when any of <code>uploaded</code>, <code>downloaded</code>, <code>result</code>, or
+      <code>failurereason</code> change.</td>
+    </tr>
+  </tbody>
+</table>
+
+### Tracking progress
+
+This can be done via the `progress` event. Remember that `downloadTotal` is whatever value you
+provided, or `0` if you didn't provide a value.
+
+    bgFetch.addEventListener('progress', () => {
+      // If we didn't provide a total, we can't provide a %.
+      if (!bgFetch.downloadTotal) return;
+
+      const percent = Math.round(bgFetch.downloaded / bgFetch.downloadTotal * 100);
+      console.log(`Download progress: ${percent}%`);
+    });
+
+### Getting the requests and responses
+
+Caution: In Chrome's current implementation you can only get the requests and responses during
+`backgroundfetchsuccess` and `backgroundfetchfailure` service worker events (see below). In future
+you'll be able to get in-progress fetches.
+
+    bgFetch.match('/podcast.mp3').then(async (record) => {
+      if (!record) {
+        console.log('No record found');
+        return;
+      }
+
+      console.log(`Here's the request`, record.request);
+      const response = await record.responseReady;
+      console.log(`And here's the response`, response);
+    });
+
+`record` is a `BackgroundFetchRecord`, and it looks like this:
+
+<table class="responsive">
+  <tbody>
+    <tr><th colspan=2>Properties</th></tr>
+    <tr>
+      <td><code>request</code></td>
+      <td><code>Request</code><br>The request that was provided.</td>
+    </tr>
+    <tr>
+      <td><code>responseReady</code></td>
+      <td><code>Promise&lt;Response&gt;</code><br>The fetched response.
+      <p>The response is behind a promise because it may not have been received yet. The promise
+      will reject if the fetch fails.</p></td>
+    </tr>
+  </tbody>
+</table>
+
+### Service worker events
+
+<table class="responsive">
+  <tbody>
+    <tr><th colspan=2>Events</th></tr>
+    <tr>
+      <td><code>backgroundfetchsuccess</code></td>
+      <td>Everything was fetched successfully.</td>
+    </tr>
+    <tr>
+      <td><code>backgroundfetchfailure</code></td>
+      <td>One or more of the fetches failed.</td>
+    </tr>
+    <tr>
+      <td><code>backgroundfetchabort</code></td>
+      <td>One or more fetches failed.
+      <p>This is only really useful if you want to perform clean-up of related data.</p></td>
+    </tr>
+    <tr>
+      <td><code>backgroundfetchclick</code></td>
+      <td>The user clicked on the download notification.</td>
```

Comment:
Do we really want to use the word "notification" here? On desktop we'd display a normal entry in the downloads bar, and other user agents could do whatever they want.